src/transformers/models/rag/modeling_tf_rag.py

```diff
@@ -1256,6 +1256,8 @@ def extend_enc_output(tensor, num_beams=None):
                 eos_token_id=eos_token_id,
                 forced_bos_token_id=None,
                 forced_eos_token_id=None,
+                supress_tokens=None,  # Not used for RAG, were added after the whisper pull request
+                begin_supress_tokens=None,  # Not used for RAG, were added after the whisper pull request
```

Comment:
I think this is not needed anymore as the default were set to `None` in the generate function